### PR TITLE
LUN-255: Back button doing nothing in vocabulary list item

### PR DESCRIPTION
**Pull request by f1sh1918**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

**Review by klinzo — APPROVED** [review]
Tested on Android.

File: src/components/ConfirmationModal.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ const ConfirmationModal = (props: ConfirmationModalProps): JSX.Element => {
   const closeModal = (): void => setVisible(false)
 
   return (
-    <Modal testID='modal' visible={visible} transparent animationType='fade'>
+    <Modal testID='modal' visible={visible} transparent animationType='fade' onRequestClose={() => setVisible(false)}>
```
Comment (SteffiStuffel) [inline src/components/ConfirmationModal.tsx:54]: Not sure if this is really the wanted behaviour. When now being in an exercise and click back-button, the modal opens, clicking back-button again the modal closes. Shouldn't the back-button then confirm the back-navigation?

Comment (klinzo) [reply src/components/ConfirmationModal.tsx:54]: I think this is fine. Its the same behavior as other apps with confirmation dialogs.

**Review by SteffiStuffel — APPROVED** [review]

File: release-notes/unreleased/LUN-255-back-button-doing-nothing-in-vocabulary-list-item.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+issue_key: LUN-255
+show_in_stores: true
+platforms:
+  - android
+  - ios
+de: Hardware Back Button schließt Modale.
```
[the remark this answers is not in the document]
Comment (f1sh1918) [reply release-notes/unreleased/LUN-255-back-button-doing-nothing-in-vocabulary-list-item.yml:6]: gonna hide it

**Pull request merged**